tests/menelaus/test_example_data.py

```diff
@@ -1,29 +1,69 @@
 from numpy import float64, int32, int64
-from menelaus.datasets import make_example_batch_data
+from menelaus.datasets.make_example_data import (
+    make_example_batch_data,
+    fetch_circle_data,
+    find_git_root,
```

Comment:
Another reason I recommend separating `find_git_root` from the dataset stuff, is to have its own test file. In that sense we later have more control over excluding it from a certain pytest configuration (i.e. if we think, do we really need to test this kind of function as frequently?).

Reply:
If cheap we can leave it in and consider this resolved -- I need to raise an issue later about `@pytest.mark.no_cover` messing with jobs if you every add `--no-cov` to your configuration from the command line.